src/components/About.js

```diff
@@ -3,13 +3,42 @@
 import React from 'react'
 
 import UsaMap from './UsaMap'
+import ucr from '../util/ucr'
+import usa, { data as usaData } from '../util/usa'
+
+const colorFromUcr = info => {
+  if (info.srs && !info.nibrs) return 'fill-blue-lighter'
+  else if (!info.srs && info.nibrs) return 'fill-blue'
+  else if (info.srs && info.nibrs) return 'fill-blue-light'
+  return 'fill-red-bright'
+}
+
+const stateColors = Object.keys(usaData).map(k => {
+  const stateName = usa(k)
+  const ucrInfo = ucr(stateName)
+  return {
+    state: k,
+    color: colorFromUcr(ucrInfo),
+  }
+}).reduce((accum, next) => ({
+  ...accum,
+  [next.state]: next.color,
+}), {})
+
+const colorCounts = Object.keys(stateColors)
+  .map(c => stateColors[c])
+  .reduce((accum, next) => {
+    const count = accum[next]
+    if (count >= 0) return { ...accum, [next]: count + 1 }
+    return { ...accum, [next]: 0 }
+  }, {})
 
 const legend = [
-  { count: 16, color: '#324D5F', text: 'Incident data only' },
-  { count: 18, color: '#95AABC', text: 'Incident and Summary data' },
-  { count: 14, color: '#DFE6ED', text: 'Summary data only' },
-  { count: 1, color: '#F48E88', text: 'Establishing an incident data only program' },
-  { count: 1, color: '#FF5E50', text: 'Currently no UCR program' },
+  { count: colorCounts['fill-blue'], color: '#324D5F', text: 'Incident data only' },
```

Comment:
I'll admit that this is less than ideal, since it uses a color which has no apparent relationship to one of the data submission categories.